src/System.Console/tests/SyncTextReader.cs

```diff
@@ -0,0 +1,206 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Xunit;
+
+public class SyncTextReaderTests
+{
+    // NOTE: These tests test the underlying SyncTextReader by
+    // accessing the Console.In TextReader (which in fact is a SyncTextReader).
+
+    static readonly string[] s_testLines = new string[] {
```

Comment:
I "stole" them from the ReadAndReadLines tests. Should probably make them
public there or extract them to a TestConstants file.
On Mar 10, 2015 4:26 PM, "Stephen Toub" <notifications@github.com> wrote:

> In src/System.Console/tests/SyncTextReader.cs
> <https://github.com/dotnet/corefx/pull/1114#discussion_r26131533>:
>
> > @@ -0,0 +1,206 @@
> > +﻿// Copyright (c) Microsoft. All rights reserved.
> > +// Licensed under the MIT license. See LICENSE file in the project root for full license information.
> > +
> > +using System;
> > +using System.IO;
> > +using Xunit;
> > +
> > +public class SyncTextReaderTests
> > +{
> > +    // NOTE: These tests test the underlying SyncTextReader by
> > +    // accessing the Console.In TextReader (which in fact is a SyncTextReader).
> > +
> > +    static readonly string[] s_testLines = new string[] {
>
> How did you arrive at this particular set of lines?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dotnet/corefx/pull/1114/files#r26131533>.
>